app/models/miq_report/generator.rb

```diff
@@ -792,9 +792,8 @@ def queue_report_result(options, res_opts)
     _log.info("Adding generate report task to the message queue...")
     task = MiqTask.create(:name => "Generate Report: '#{name}'", :userid => options[:userid])
 
-    MiqQueue.put(
-      :queue_name  => "reporting",
-      :role        => "reporting",
```

Comment:
Note, this was probably not causing the issue seen above because it specified the role as reporting, which should set the zone to nil.  To simplify things, I moved this code to use the `submit_job` method to be more consistent.